src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx

```diff
@@ -176,71 +182,81 @@ export default function SelectPurposeForm(props: SelectPurposeFormProps): JSX.El
+                  </RadioGroup>
+                </FormControl>
+              </Grid>
 
-            <Grid display='flex'>
-              {batch && (
+              <Grid display='flex'>
                 <Grid item xs={isNurseryTransfer ? 6 : 12} sx={{ marginTop: theme.spacing(2) }} paddingRight={1}>
                   <Dropdown
                     id='from'
                     label={strings.FROM_NURSERY}
-                    selectedValue={batch.facilityId.toString()}
-                    options={[getNurseriesById(organization, batch.facilityId)].map((nursery) => ({
+                    selectedValue={firstBatch.facilityId.toString()}
+                    options={[getNurseriesById(organization, firstBatch.facilityId)].map((nursery) => ({
                       label: nursery.name,
                       value: nursery.id.toString(),
                     }))}
-                    onChange={(value) => onChange('destinationFacilityId', value)}
+                    onChange={() => true}
                     fullWidth={true}
                   />
                 </Grid>
-              )}
 
-              {isNurseryTransfer && (
-                <Grid item xs={6} sx={{ marginTop: theme.spacing(2) }} paddingLeft={1}>
-                  <Dropdown
-                    id='destinationFacilityId'
-                    label={strings.TO_NURSERY_REQUIRED}
-                    selectedValue={record.destinationFacilityId?.toString()}
-                    options={getAllNurseries(organization).map((nursery) => ({
-                      label: nursery.name,
-                      value: nursery.id.toString(),
-                    }))}
-                    onChange={(value) => onChange('destinationFacilityId', value)}
-                    errorText={fieldsErrors.destinationFacilityId}
-                    fullWidth={true}
-                  />
-                </Grid>
-              )}
-            </Grid>
-            <Grid item xs={6} sx={{ marginTop: theme.spacing(2) }}>
-              <DatePicker
-                id='withdrawnDate'
-                label={strings.WITHDRAW_DATE_REQUIRED}
-                aria-label={strings.WITHDRAW_DATE_REQUIRED}
-                value={record.withdrawnDate}
-                onChange={onChangeDate}
-                errorText={fieldsErrors.withdrawnDate}
-              />
-            </Grid>
-            <Grid item xs={12} sx={{ marginTop: theme.spacing(2) }}>
-              <Textfield id='notes' value={record.notes} onChange={onChange} type='textarea' label={strings.NOTES} />
+                {isNurseryTransfer && (
+                  <Grid item xs={6} sx={{ marginTop: theme.spacing(2) }} paddingLeft={1}>
+                    <Dropdown
+                      id='destinationFacilityId'
+                      label={strings.TO_NURSERY_REQUIRED}
+                      selectedValue={firstBatch.destinationFacilityId?.toString()}
+                      options={getAllNurseries(organization).map((nursery) => ({
```

Comment:
Missed this. The list should only be nurseries that the batches belong to.
This feature is primarily for narrowing down nursery for batches coming from inventory page.

Reply:
For batches coming from details page this will end up being a list of one nursery

Reply:
Hence auto selected / defaulted to that value

Reply:
But if we have selected only one batch and we want to do a "Nursery Transfer", would it be "from" the nursery it belongs "to" the nursery it belongs?

Reply:
I ended up commenting on the wrong code, I was referring to the list of 'from nurseries' - which you have already fixed. my bad!!

Reply:
btw do we may need to filter out the from nursery from this list? we can do that as a follow up

Reply:
Correct, I will do this in a following PR